src/App.js

```diff
@@ -1,10 +1,30 @@
-import React from 'react';
+// Import Dependencies
+  import React from 'react';
+    // Import Router
+    import { Route } from 'react-router-dom';
+
+// Import Components
+  import { 
+    Home, 
+    About, 
+    Contact, 
+    Navigation 
+  } from './components';
+
+// Styles
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+
+
+
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path="/"         exact   component={Home}    />
+    <Route path="/about"    exact   component={About}   />
+    <Route path="/Contact"  exact   component={Contact} />
```

Comment:
Tiny change - URLs are typically lowercase 
```suggestion
    <Route path="/contact"  exact   component={Contact} />
```
Also, no need to use `exact` on every `<Route />`, only on your `'/'` route in this exercise